### PR TITLE
contrib/fonts-nerd-fonts: new package (3.2.1)

### PR DESCRIPTION
**Pull request by jcgruenhage**
This package is quite a mouthful, but I'm not sure how to improve this. Would be happy to receive feedback on this.

### CODE REVIEW

File: contrib/fonts-nerd-fonts/template.py [hunk 1]
```diff
@@ -0,0 +1,479 @@
...
+        self.license = spdx
+        return [
+            f"usr/share/fonts/nerd-font-{package}",
+            f"usr/share/licenses/fonts-nerd-font-{package}",
+        ]
+
+
+for _font in _fonts:
+    _font_subpackage(
+        _font["package"], _font["name"], _font["folder"], _font["spdx"]
```
Comment (jcgruenhage) [inline contrib/fonts-nerd-fonts/template.py:478]: expand with `*_font` after switching to tuples

**Review by jcgruenhage — COMMENTED** [review]
feedback from IRC

**Comment by jcgruenhage** [comment]
This is a duplicate of https://github.com/chimera-linux/cports/pull/1961, but I'd argue that my approach makes maintenance easier.

File: contrib/fonts-nerd-fonts/template.py [hunk 2]
```diff
@@ -0,0 +1,208 @@
...
+    "b7a3eee24d2b0910f3a07705f1e053221d2535f07c5a13614c4dead2620e9978",
+    "658cc9506acf89b5f5dfc6b55eb9899b782dcf8d89f3d11c9abb2c4d52617f12",
+    "21b43c2314614733f5e9b4b4f3117edc3986d2b0f2a33409b111cc38360e8391",
+    "aa1b09dca861a306cd24c2c47fded88b63ae833ff0e815dad3642465583e300f",
+    "2df9c9e54e5a2bc19c2ffd1df7ee283e02ba2e9a5fda1067b4884375ae4b0b7a",
+]
+options = ["empty"]
+
+
+def do_install(self):
```
Comment (q66) [inline contrib/fonts-nerd-fonts/template.py:78]: the functions should go below the _fonts definition

File: contrib/fonts-nerd-fonts/template.py [hunk 3]
```diff
@@ -0,0 +1,208 @@
...
+    for font in _fonts:
+        folder = font[2]
+        package = font[0]
+        self.install_file(
+            f"{folder}/*.*tf",
+            f"usr/share/fonts/nerd-{package}",
+            glob=True,
+        )
+        for file in [
```
Comment (q66) [inline contrib/fonts-nerd-fonts/template.py:87]: not sure if it's a good idea to install all the licenses separately for all fonts

Comment (jcgruenhage) [reply contrib/fonts-nerd-fonts/template.py:87]: Well separate fonts have separate licenses, how would you do it instead?

Comment (q66) [reply contrib/fonts-nerd-fonts/template.py:87]: nevermind, i just misread what the code does - you have it right

**Pull request merged**